magellan-flow-sample/src/androidTest/java/com/wealthfront/magellan/assertions/AssertShown.kt

```diff
@@ -0,0 +1,61 @@
+package com.wealthfront.magellan.assertions
+
+import android.view.View
+import androidx.annotation.StringRes
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.ViewAssertion
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.Visibility
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withClassName
+import androidx.test.espresso.matcher.ViewMatchers.withEffectiveVisibility
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.espresso.util.HumanReadables
+import kotlin.reflect.KClass
+import org.hamcrest.CoreMatchers.allOf
+import org.hamcrest.CoreMatchers.containsString
+import org.hamcrest.CoreMatchers.equalTo
+import org.hamcrest.Matcher
+
+fun assertShown(action: AssertionBuilder.() -> Unit) {
```

Comment:
Oh cool, this works out of the box with robolectric? Or is this actually used in espresso?

Reply:
im just using this with espresso for now. But we can make a similar version that works with robolectric, i think